index.bs

```diff
@@ -517,6 +526,19 @@ that it's easy for the user to input PSK on the device.  Supported PSK input met
 are numeric and scanning a QR-code.  Devices with non-zero PSK ease of input must
 support the numeric PSK input method.
 
+Any authentication method may require an `auth-initation-token` before
+showing a PSK to the user or requesting PSK input from the user.  If an
+advertising agent has the `pw` field in its mDNS TXT record, it must be used as
+the `auth-initation-token` in the the first authentication message sent to
+or from that agent.  Agents should discard any authentication message whose
+`auth-initation-token` is set and does not match the `pw` provided by the
+advertising agent.
+
+TODO: Autolink advertising agent when PR #180 lands.
```

Comment:
Nit: TODOs should have a bug or an alias.

Reply:
Resolved this TODO.